deployments/xpumanager_sidecar/kustomization.yaml

```
@@ -1,5 +1,5 @@
 resources:
-- https://raw.githubusercontent.com/intel/xpumanager/V1.2.18/deployment/kubernetes/daemonset-intel-xpum.yaml
+- https://raw.githubusercontent.com/intel/xpumanager/V1.2.29/deployment/kubernetes/daemonset/base/daemonset-intel-xpum.yaml
```

one small nit: they provide kustomization also so it would be possible to do something similar as we have with deployments/nfd/base/ kustomization:
```suggestion
- https://github.com/intel/xpumanager/deployment/kubernetes/daemonset/base/?ref=V1.2.29
```

---

Updated